web/themes/joinup/templates/rdf_entity/rdf-entity--asset-release--full.html.twig

```diff
@@ -7,7 +7,7 @@
 <article{{ attributes }}>
   <div class="page__content">
     {{ content.field_isr_release_notes }}
-    <div class="version-wrapper">{{ content.field_isr_is_version_of }}</div>
+    <div class="fieldset fieldset--version-wrapper">{{ content.field_isr_is_version_of }}</div>
```

Comment:
`fieldset--version-wrapper` seems unnecessary and also too specific. Not to speak about the fact there is already a `fieldset` wrapper ...